cmd/ufs/ufs.go

```diff
@@ -1,13 +1,21 @@
-// This is a ufs server.
+// This is a userspace server which exports a filesystem over 9p(2000).
+//
+// UFS exports a filesystem from the server running it via 9p over
```

Comment:
Sure, I took it out. I was treating the first line as a subject and the second as a paragraph, but I guess they do say the same thing and the second one is only a sentence long anyways.